app/static/index.css

```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Dosis&family=Poppins:wght@300&family=Roboto:wght@300&display=swap');
+
```

Comment:
- [ ] Great job so far with the desktop version. Actually, this project is for you to build the mobile section first. Please correct that. You can check that by clicking on inspect and choosing the mobile version.
Your mobile
![Screenshot (300)](https://user-images.githubusercontent.com/65298048/185359551-44d46ecb-5a89-484a-8f38-7e59d0164def.png)
figma mobile
![Screenshot (301)](https://user-images.githubusercontent.com/65298048/185359746-dd9ff65a-28f4-4562-bb42-9ed260284f38.png)